packages/apollos-data-prayer/src/prayer-requests/data-source.js

```diff
@@ -102,17 +103,19 @@ export default class PrayerRequest extends RockApolloDataSource {
         LastName,
         Text, // Required by Rock
         CategoryId,
-        CampusId,
+        CampusId: parseInt(parseGlobalId(CampusId).id, 10),
         IsPublic: true,
         RequestedByPersonAliasId: primaryAliasId,
         IsActive: true,
         IsApproved: true,
         EnteredDateTime: new Date().toJSON(), // Required by Rock
       });
       // Sets the attribute value "IsAnonymous" on newly created prayer request
+      // TODO: we should combine this so network doesn't die and someone's prayer is left un-anonymous
```

Comment:
What are you thinking here?

Reply:
We should set the set the attribute to anonymous on the same post call we used to make the create the prayer. that way if there's a network failure in between calls, someone's prayer won't be left public.